kogito-cloud-operator.jenkinsfile

```diff
@@ -10,18 +52,54 @@ pipeline {
         MAVEN_RELEASE_VERSION = "3.6.3"
         GRAALVM_RELEASE_VERSION = "19.3.1"
         OPERATOR_SDK_RELEASE_VERSION = "v0.15.1"
+
+        // Just for reference, env variables we are setting in `Initialize` step
+        // keep commented as it needs to be changed (if set here, then it is unmutable ...)
+        // TEST_PARAMS = "essai"
+        // OPENSHIFT_API_URL = ""
+        // INTERNAL_IMAGE_REGISTRY = ""
+        // EXTERNAL_IMAGE_REGISTRY = ""
     }
     
     stages {
         stage("Init"){
             steps{
-                script {
-                    sh """
-                        podman login -u="${REGISTRY_USER}" -p="\${REGISTRY_PASSWORD}" ${IMAGE_REGISTRY}
-                    """
-                    
+                script {                    
                     // Calculate build display name
                     setBuildDisplayName()
+
+                    // Set Openshift & registry urls
+                    env.OPENSHIFT_API_URL = "https://api.${params.OPENSHIFT_URL_DOMAIN}:6443"
+                    if (params.USE_OPENSHIFT_INTERNAL_REGISTRY){
+                        env.INTERNAL_IMAGE_REGISTRY = "image-registry.openshift-image-registry.svc:5000"
+                        env.EXTERNAL_IMAGE_REGISTRY = "default-route-openshift-image-registry.apps.${params.OPENSHIFT_URL_DOMAIN}"
+                    } else {
+                        env.INTERNAL_IMAGE_REGISTRY = "${params.IMAGE_REGISTRY}"
+                        env.EXTERNAL_IMAGE_REGISTRY = "${params.IMAGE_REGISTRY}"
+                    }
+
+                    // Test params
+                    env.TEST_PARAMS=""
+                    if (params.SKIP_BUILD) {  addTestParamFromValue("cr_deployment_only", "true") }
```

Comment:
What is the purpose of this line?

Reply:
Yeah I should add a comment ...
If skip_build, then we have no CLI, so we need to use CR only in tests